projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -53,6 +53,9 @@ function wpcom_add_wpcom_menu_item() {
 	if ( ! function_exists( 'wpcom_is_nav_redesign_enabled' ) || ! wpcom_is_nav_redesign_enabled() ) {
 		return;
 	}
+	if ( function_exists( 'is_agency_managed_site' ) && is_agency_managed_site() ) {
+		return;
+	}
```

Comment:
Hides the "Hosting drawer."

<img src="https://private-user-images.githubusercontent.com/5634774/341844871-5bf481b2-aae0-452f-955a-a3e312d84238.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkwMDY1NjksIm5iZiI6MTcxOTAwNjI2OSwicGF0aCI6Ii81NjM0Nzc0LzM0MTg0NDg3MS01YmY0ODFiMi1hYWUwLTQ1MmYtOTU1YS1hM2UzMTJkODQyMzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMjE0NDI5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWEzMDBiNmUxNDdhMGQxYjQ2OGExN2FmN2Q5M2I2OWIyMzI4NzkwNzI2ZmRlNzdhYjFmMWQ0ZjQxZmRkN2MyYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.tIk_ob64TwRC6EsV72vBzHK3YKAyvErShBRNPBMeAuc" />